docs/src/pages/start-here.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Start Here
+---
+
+# Start here
+
+## What is this?
+
+[These organizations](about) are working together to define a common set of
+annotation types for use in Java static analysis, beginning with nullness
```

Comment:
Maybe we should be a little more explicit here about the fact that other domains are coming?

Reply:
It seems explicit about that to me.  Static analysis annotations, *starting with*.

What it doesn't hint at at all is the other things on wiki/long-term-roadmap.